webapi/src/main/scala/org/knora/webapi/util/InputValidation.scala

```diff
@@ -54,20 +54,21 @@ object InputValidation {
     val PrecisionSeparator: String = "-"
 
     // The expected format of a Knora date.
-    // Calendar:YYYY[-MM[-DD]][:YYYY[-MM[-DD]]]
+    // Calendar:YYYY[-MM[-DD]][ EE][:YYYY[-MM[-DD]][ EE]]
+    // EE being the era: one of BC or AD
     private val KnoraDateRegex: Regex = ("""^(GREGORIAN|JULIAN)""" +
         CalendarSeparator + // calendar name
         """\d{1,4}(""" + // year
         PrecisionSeparator +
         """\d{1,2}(""" + // month
         PrecisionSeparator +
-        """\d{1,2})?)?(""" + // day
+        """\d{1,2})?)?( BC| AD| BCE| CE)?(""" + // day
```

Comment:
The convention for writing date with era is date+whitespace+BC/AD/BCE/CE I would say  let's leave the whitespace before the era